security/pass-otp/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        tadfisher pass-otp 1.1.1 v
+maintainers         {loicp.eu:loic-github @lpefferkorn}
+categories          security
+description         A pass extension for managing one-time-password (OTP) tokens
+long_description    ${description}
```

Comment:
We prefer to not simply copy the short description. It's not a blocker, but can the `long_description` be expanded?